internal/pkg/describe/uri.go

```diff
@@ -220,37 +220,41 @@ type albDescriber struct {
 	svcDescriber    ecsDescriber
 	envDescriber    envDescriber
 	initLBDescriber func(string) (lbDescriber, error)
-	envDNSNameKey   string
+	envDNSTypeKey   string
 }
 
-func (d *albDescriber) envDNSName(path string) (albURI, error) {
+func (d *albDescriber) envDNSName(path string) (publicURI, error) {
 	envOutputs, err := d.envDescriber.Outputs()
 	if err != nil {
-		return albURI{}, fmt.Errorf("get stack outputs for environment %s: %w", d.env, err)
+		return publicURI{}, fmt.Errorf("get stack outputs for environment %s: %w", d.env, err)
 	}
-	return albURI{
-		DNSNames: []string{envOutputs[d.envDNSNameKey]},
+	dnsNames := []string{envOutputs[d.envDNSTypeKey]}
+	if cfDNS, ok := envOutputs[envOutputCloudFrontDomainName]; ok {
```

Comment:
Do we display only the CloudFrontDomainName is the security group ingress is restricted to cloudfront? Is there a way of achieving that 🥺

Reply:
Yes there is! I'm planning on doing this in one more PR because the PrefixList PR isn't merged yet.